src/sd-helper.c

```diff
@@ -31,22 +31,30 @@
  *
  * @param[in] source sd_event_source that should be prepared.
  * @param[in] timeout not used
+ * @return gboolean, TRUE on success, FALSE otherwise
  * @see https://developer.gnome.org/glib/stable/glib-The-Main-Event-Loop.html#GSource
  */
 static gboolean sd_source_prepare(GSource *source, gint *timeout)
 {
-        return sd_event_prepare(((struct SDSource *) source)->event) > 0 ? TRUE : FALSE;
+        g_return_val_if_fail(source, FALSE);
+
+        return sd_event_prepare(((struct SDSource *) source)->event) > 0
```

Comment:
Did we enforce line breaks anywhere? Personally, I find the first version more readable.
In RAUC we also do not cut hard at 80 chars

Reply:
Not yet. I'm in favor of wrapping long lines > 100 characters (instead of 80) though, because:
- nested function calls containing memory leaks and wrong error handling become more apparent
- functions with too many arguments become apparent
- code readability
- side by side code view, even on smaller (e.g. laptop) screens

I'll push the 100 characters per line limit.

Reply:
Ok for me